mobile/src/calendar/days/days.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { styles, colors } from '../styles';
+import { Tile, EmptyTile } from './tile/tile';
+import { TileSeparator } from './tile/tile-separator';
+import { Days as DaysModel } from '../../reducers/calendar/days.model';
+import { AppState } from '../../reducers/app.reducer';
+import { Dispatch, connect } from 'react-redux';
+import { CalendarActions, loadDays, LoadDays } from '../../reducers/calendar/calendar.action';
+import { bindActionCreators } from 'redux';
+
+interface DaysProps {
+    days: DaysModel;
+}
+
+interface DaysDispatchProps {
+    loadDays: () => void;
+}
+
+class DaysImpl extends Component<DaysProps & DaysDispatchProps, {}> {
+    public componentDidMount() {
+        this.props.loadDays();
+    }
+
+    public render() {
+        const { vacation, off, sick } = this.props.days;
+
+        const vacationTile = vacation
+            ? <Tile leftDays={vacation.leftDays} allDays={vacation.allDays} title={vacation.title} leftColor={colors.days.left} allColor={colors.days.all} />
```

Comment:
One thing I'm not sure I like is `Tile` name. It is too generic, and if we add tiles somewhere else we'll be confused while importing things... On the other hands, they are encapsulated inside the folder and having DaysTile / DaysCounter/ DaysTileIndicator is very messy. Do not change anything unless you have a better idea :)

Reply:
I relied on folder structure, it would separate those generic named components, but I'm not sure too. Maybe that:
````
DaysCounters\ (agree, DaysCounters sounds good)
     DaysCounter.tsx
     DaysCounterIndicator.tsx
     DaysCounterSeparator.tsx
````